omod/src/main/webapp/pages/htmlform/viewEncounterWithHtmlForm.gsp

```diff
@@ -36,7 +36,13 @@
 <% } %>
 
 <span id="form-actions" class="no-print">
-    <a class="button" id="print-button" href="javascript:window.print()">
+
+<% if (customPrintPageName == null || customPrintPageName.isEmpty()) { %>
+        <a class="button" id="print-button" href="javascript:window.print()">
+<% } else { %>
+        <a class="button" id="print-button" href="${ui.pageLink(customPrintProvider, customPrintPageName, [patientId: patient.patient.id, encounterUuid: encounter.uuid, target: customPrintTarget])}" target="${customPrintTarget}">
```

Comment:
Would it be worth adding the rel attribute/values: `rel="noreferrer noopener"` per MDN suggestion here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target